### ISSUE TITLE
Reverent bugs.

### ISSUE THREAD

**Issue by Nakijimakai** [ParadiseSS13/Paradise#6098]
**Problem Description**:
Reverent Night vision still doesn't work, can suffocate out in space, I was unable to purchase Malfunction even when I had over 600 essence and the health bar does not work i went from 100% Hp to 0% with no numbers in-between
https://github.com/ParadiseSS13/Paradise/issues/new

**What did you expect to happen**: 
Night vision to work, a Ghost to not suffocate in space, Malfunction should available to purchase having a quantifiable health bar.

**What happened instead**:
No night vision, I started suffocating out in space, I was told i did not have enough stolen essence when I had over 600 essence, Health bar stayed at 100% till death.

**Why is this bad/What are the consequences:**
When people knock out the lights it is hard to follow people in the dark, Rev in ghost form should not breath in space, a major skill is not able to be purchased, having an idea of what your health situation is supposed to be is nice.

**Steps to reproduce the problem**:
wait till mid-round antag spawn get reverent.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
Game mode is changeling, was being ghost before called for reverent, time was 3:10AM MST

**Comment by Alexshreds** [comment]
Buying malfunction does work, you don't buy it with "stolen essence" as that's just the total you've accumulated and stays after spending normal essence.

**Comment by variableundefined** [comment]
If this happened, it has been fixed
